Gopkg.lock

```diff
@@ -653,8 +653,8 @@
     "prometheus/testutil",
   ]
   pruneopts = "UT"
-  revision = "abad2d1bd44235a26707c172eab6bca5bf2dbad3"
-  version = "v0.9.1"
+  revision = "505eaef017263e299324067d40ca2c48f6a2cf50"
+  version = "v0.9.2"
```

Comment:
Is there anything in particular in v0.9.2 that we require? I don't have an objection to bumping the dependency version, but it'd be good to understand why 😄

Reply:
🤦‍♂️ just read your PR description! Resolving...